soda/core/soda/common/attributes_handler.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+from soda.common.logs import Logs
+
+
+class AttributeHandler:
+    def __init__(self, logs: Logs) -> None:
+        self.logs = logs
+
+    def validate(self, attributes: dict[str, any], schema: list[dict]) -> tuple[dict[str, any]]:
+        "Validates given attributes against given schema. Returns a dict with valid and a dict with invalid attributes."
```

Comment:
Did you mean this to be a docstring? In that case, please use """

Reply:
uh yes, fixed.